readme.md

```diff
@@ -110,6 +110,7 @@ Thanks for contributing! 🦋🙌
 - [](# "file-finder-buffer") [Lets you start typing your search immediately after invoking the File Finder (<kbd>t</kbd>), instead of having you wait for it to load first.](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)
 - [](# "preserve-file-finder-term") Preserves the search terms when navigating back and forth between the File Finder and the files.
 - [](# "show-associated-branch-prs-on-fork") [Shows the associated pull requests on branches for forked repositories.](https://user-images.githubusercontent.com/16872793/81504659-7e5ec800-92b8-11ea-9ee6-924110e8cca1.png)
+- [](# "rgh-feature-descriptions") [Shows the description and screenshot of each Refined GitHub feature when visiting the features’ file.](https://user-images.githubusercontent.com/202916/103462909-e4f19400-4d28-11eb-86d3-5b97d75b4781.png)
```

Comment:
> Since this feature is enabled exclusively on very specific files on our repo, it neither needs to be documented nor should it be disableable. 
```suggestion
```

Reply:
The feature needs to be excluded in `webpack.config.ts` so it doesn't error